presto-docs/src/main/sphinx/optimizer/merge-join-optimizations.rst

```diff
@@ -0,0 +1,30 @@
+========================
```

Comment:
The documentation is great, but I think we should wait to add it until the feature is ready to use to before adding it.